deploy-board/deploy_board/templates/clusters/cluster_configuration.html

```diff
@@ -464,7 +464,7 @@ <h4 class="panel-title pull-left pointer-cursor">
                         }).filter(function(item){return item.name!='assign_public_ip'}),
                 assignPublicIP: initialAssignPublicIP,
                 awsRole: currentCluster.configs.aws_role,
-                baseimages: baseImageSorted.map(function(o) {
+                baseimages: info.baseImages.map(function(o) {
```

Comment:
https://github.com/pinterest/teletraan/blob/a164d2e7750b98dd59eec139f0b2036aaae2a74b/deploy-board/deploy_board/templates/clusters/cluster_configuration.html#L445

Looks like this change will no longer filter for accepted images?

Reply:
we don’t show accepted images in cluster creation page. So, to keep creation and configuration consistent, I plan to make them the same

Reply:
I am not sure why there are some code to filter accepted and sort the AMIs. In creation page, it just show all the AMIs.